2018-02-12 17:58
['43', '0.5', '2', '30', '']
srun: error: jupiterace: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "paralellmultitester.py", line 32, in num_order
    f=list(map(float,f))
ValueError: could not convert string to float: 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "paralellmultitester.py", line 38, in <module>
    for filein in sorted(glob.glob(path + "*.model"),key=num_order,reverse=True):
  File "paralellmultitester.py", line 35, in num_order
    assert False
AssertionError
